ansible/files/bootstrap/telemetry.sh

```diff
@@ -142,6 +142,9 @@ aws s3 cp $TELEMETRY_CONF_BUCKET/sbt-$sbt_version.rpm $HOME/sbt.rpm
 sudo yum -y localinstall $HOME/sbt.rpm
 rm $HOME/sbt.rpm
 
+# Install spark packages
+sudo aws s3 cp --recursive $TELEMETRY_CONF_BUCKET/spark-packages/ /usr/lib/spark/jars/
```

Comment:
I've staged this file in my personal dev bucket:

https://console.aws.amazon.com/s3/object/telemetry-spark-emr-2-dev-klukas/spark-packages/mozilla-assembled-spark-packages.jar

We could copy from there into the stage and prod buckets for now, but I'd like a reproducible way to generate and upload that jar.

It looks like there's some other content in the S3 bucket that isn't generated by this repo. Where do those other files come from?

We could potentially add an ansible task that does a `docker run` with our mozilla:sbt container to build the jar, but how to make that happen is above my knowledge of ansible.

Are there other possibilities routes to take here?

Reply:
I would prefer to somehow pull the jar for the s3 maven repo

Reply:
I don't think an assembly jar that includes all dependencies is a good fit for uploading to the mavenrepo, but we could certainly stage a jar in the artifacts s3 bucket.

This might look like creating a new repo (mozilla/telemetry-spark-assembly perhaps?) that just consists of a build.sbt and a .circleci/config.yml that runs `sbt assembly` and uploads to the artifacts bucket when the repo is tagged.

Reply:
ooooh, it's an assembly, my bad

Reply:
See https://github.com/jklukas/telemetry-spark-packages-assembly/commit/fda20323a8b662799e590a38b8b7813bc7cb8057

Then, we'd pull down the assembly jar directly from a path in the artifacts bucket. Are you good with that approach, @relud?